### DISCUSSION TITLE
hardware/co/2

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#14] [Announcements]
# hardware/co/2

NoughtQ的笔记本

https://note.noughtq.top/hardware/co/2

**Comment by fang88666** [comment]
4 Instruction Representations中 UJ-type似乎不能用于`jalr`，是不是笔误呢？

**Reply by NoughtQ** [reply]
雀食，因为jal和jalr都是无条件分支指令，所以第一次学的时候我误以为jalr也是UJ型指令——它应该是**I型指令**，等会儿我会改一下，感谢指正 ♪(･ω･)ﾉ
